jib-maven-plugin/src/main/java/com/google/cloud/tools/jib/maven/JibPluginConfiguration.java

```diff
@@ -143,6 +171,23 @@ public void set(String image) {
     @Nullable @Parameter private String user;
   }
 
+  /** Configuration for the {@code extraDirectory} parameter. */
+  public static class ExtraDirectoryParameters {
+
+    @Nullable @Parameter private File path;
+
+    @Parameter private List<PermissionConfiguration> permissions = Collections.emptyList();
+
+    public void set(File path) {
```

Comment:
nit: maybe add a javadoc about how this allows for setting a path string directly as the `extraDirectory`